### ISSUE TITLE
Unrecognized keyword argument "density" for Geom.histogram

### ISSUE THREAD

**Issue by xiaoweiz** [dcjones/Gadfly.jl#530]
Julia 0.3.5 + Gadfly 0.3.10

Running 
```python
plot(dataset("ggplot2", "diamonds"), x="Price", color="Cut", Geom.histogram(bincount=30, density=true))
```
yields
```shell
ERROR: unrecognized keyword argument "density"
```

**Comment by acheck10** [comment]
I just encountered the same issue.

**Comment by vchuravy** [comment]
This was fixed in 5b9d72fa040c45, but since then no new version of Gadfly was published.

**Comment by dcjones** [comment]
I just tagged a new version, so it should work after updating.
